src/app/components/Hero/Hero.jsx

```diff
@@ -10,17 +10,17 @@ class Hero extends React.Component {
   render() {
     return (
       <div
-        className="Hero"
+        className="hero"
         style={{ backgroundImage: `url("${this.props.heroData.heroImageUrl}")` }}
       >
-        <div className="Hero__container nypl-full-width-wrapper">
-          <div className="Hero__text nypl-column-three-quarters nypl-column-offset-one">
-            <div className="Hero__text__HeroTitle">
+        <div className="hero__container nypl-full-width-wrapper">
+          <div className="hero__text nypl-column-three-quarters nypl-column-offset-one">
+            <div className="hero__text__HeroTitle">
               {this.props.heroData.category ? <p>{this.props.heroData.category}</p> : ''}
-              <h1 className="Hero__text__HeroTitle__des">
+              <h1 className="hero__text__HeroTitle__des">
```

Comment:
`HeroTitle` is capitalize but the next one is `heroTitle`, not sure how much the CSS relies on this.